flocker/ca/_script.py

```diff
@@ -0,0 +1,112 @@
+# Copyright Hybrid Logic Ltd.  See LICENSE file for details.
+
+"""
+The command-line certificate authority tool.
+"""
+
+import os
+import sys
+
+from twisted.internet.defer import maybeDeferred, Deferred
+from twisted.python.filepath import FilePath
+from twisted.python.usage import Options, UsageError
+
+from zope.interface import implementer
+
+from ..common.script import (flocker_standard_options, ICommandLineScript,
+                             FlockerScriptRunner)
+
+from ._ca import (CertificateAuthority, CertificateAlreadyExistsError,
+                  KeyAlreadyExistsError)
+
+
+class InitializeOptions(Options):
+    """
+    Command line for ``flocker-ca initialize``.
+    """
+
+    longdesc = """Create a new certificate authority.
+
+    Creates a private/public key pair and self-signs the public key to
+    produce a new certificate authority root certificate. These are stored
+    in the current working directory. Once this has been done other
+    ``flocker-ca`` commands can be run in this directory to create
+    certificates singed by this particular certificate authority.
+
+    Parameters:
+
+    * name: Will be used as the name of the certificate authority,
+      e.g. "mycluster".
+    """
+
+    synoposis = "<name>"
```

Comment:
This synopsis doesn't seem to be included in the `--help` output.

```
(1636)(ca-initialization-FLOC-1636 ? =)[~/.../HybridLogic/flocker]$ flocker-ca initialize --help
Usage: flocker-ca <command> [OPTIONS] initialize [options]
Options:
      --version  Display Twisted version and exit.
      --help     Display this help and exit.

Create a new certificate authority.      Creates a private/public key pair and
self-signs the public key to     produce a new certificate authority root
certificate. These are stored     in the current working directory. Once this
has been done other     ``flocker-ca`` commands can be run in this directory to
create     certificates singed by this particular certificate authority.
Parameters:      * name: Will be used as the name of the certificate authority,
e.g. "mycluster".

```

And the bullet list description  of `name` isn't rendered properly.